### ISSUE TITLE
Remove generated files from GitHub

### ISSUE THREAD

**Issue by gcomte** [fluidex/orchestra#25]
Generated files (particularly the `.pb`-files which are a binaries) should not be part of the version control but created during a build process. 

There is no need to keep a version history of these files, as they can be generated at all times, from all (previous) states. In turn they using the versioning pretty messy though if you want to rebase or something, because you'll get a ton of merge conflicts in files that you don't actually care about.

**Comment by silathdiir** [comment]
Yes. The generated files should be added to `.gitignore` after we update the projects (as [dingir-exchange](https://github.com/fluidex/dingir-exchange) and [rollup-state-manager](https://github.com/fluidex/rollup-state-manager)) which are using this lib. May call this `make` command in `build.rs` of these projects.

I will create issues for associating projects.
Thanks a lot.

**Comment by HAOYUatHZ** [comment]
Yes I agree.

We should remove the binary files from history and just put it into https://github.com/fluidex/orchestra/releases. (Can use github action?)

BTW, to remove binaries from history we should follow https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github#removing-a-file-that-was-added-in-an-earlier-commit using `bfg` or `git filter-branch`.
